sources/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ target_link_libraries(dynawoAlgorithms
   $<$<NOT:$<BOOL:${USE_LEGACY_IIDM}>>:LibXml2::LibXml2>
   )
 
-install(PROGRAMS execDynawoAlgorithms.sh DESTINATION ${BINDIR_NAME})
+install(PROGRAMS dynawo-algorithms.sh DESTINATION .)
```

Comment:
This script is used in the equivalent private repository so you need to make sure everything is still working from the platform perspective

Reply:
related pr has already been done